### PR TITLE
Fixing Travis

### PR DESCRIPTION
**Pull request by wizzymore**
This PR is for fixing travis builds

For the moment it gives us some errors so we have to fix this

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
all our travis are on 18.04 now?

**Comment by wizzymore** [comment]
I think some of the modules are not

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +13,7 @@ services:
   - mysql
 
 git:
-  depth: 1
+  depth: 10
```
Comment (FrancescoBorzi) [inline .travis.yml]: why this change?

Comment (FrancescoBorzi) [inline .travis.yml]: why this change?

Comment (wizzymore) [reply .travis.yml]: The travis.yml was copied from another module, i don't understand that change either but i keeped it like on other modules

Comment (BarbzYHOOL) [reply .travis.yml]: not sure what it does, probably for some recursive stuff

Comment (wizzymore) [reply .travis.yml]: Travis is now okay, tell me if i should mark this as resolved

Comment (BarbzYHOOL) [reply .travis.yml]: yes its ok

Comment (BarbzYHOOL) [reply .travis.yml]: in fact it's the number of commits when you clone @wizzymore  (so latest commit + 10), dunno if we need 10, 1 or 0. 10 is fine tho, but just saying
